repos/c2sm/packages/icon/package.py

```diff
@@ -149,10 +152,11 @@ class Icon(AutotoolsPackage):
         default=False,
         description=
         'Enable PGI/NVIDIA cross-file function inlining via an inline library')
-    variant('nccl',
+    variant('nccl', default=False, description='Enable NCCL for communication')
+    variant('cuda-graphs',
             default=False,
-            description='Ennable NCCL for communication')
-
+            description=
+            'Enable CUDA graphs. Warning! This is an experimental feature')
```

Comment:
What is the plan with that variant? Will it become non-experimental at some point?

Reply:
Yes. The plan is to make this operational at MCH.